anti-analysis/packer/gopacker/gopacker_capa.yml

```diff
@@ -0,0 +1,29 @@
+rule:
+  meta:
+    name: packed with GoPacker
+    namespace: anti-analysis/packer/gopacker
+    author: jared.wilson@fireeye.com
+    description: The sample appears to be packed with GoPacker.
+    scope: file
+    att&ck:
+    - Defense Evasion::Obfuscated Files or Information [T1027.002]
+    mbc:
+      - Anti-Static Analysis::Software Packing
+    examples:
+      - ebfba63891471ced641b60e0403777da
+      - 03ae008aa48e52d32517a60e2ce1a1d2
+    references:
+      - https://github.com/petercunha/gopacker
+  features:
+    - and:
+        - string: "Anonymous"
+        - string: "<autogenerated>"
+        - string: "github.com/nirhaas/gopacker/"
+        - string: ".packed"
```

Comment:
is this a section name? if so, use feature `section: .packed`

Reply:
It's not, it's actually the default extension given to packed files. I'll add a Description.